### PR TITLE
travis-ci: fixed ci failed

### PR DESCRIPTION
**Pull request by syzh**

### CODE REVIEW

File: bin/resty [hunk 1]
```diff
@@ -768,10 +768,7 @@ $main_include_directives
 
 stream {
     access_log off;
-    lua_socket_log_errors off;
```
Comment (doujiang24) [inline bin/resty:771]: why do we need this change?

File: bin/resty [hunk 2]
```diff
@@ -768,10 +768,7 @@ $main_include_directives
...
     resolver @nameservers;
-    lua_regex_cache_max_entries 40960;
```
Comment (doujiang24) [inline bin/resty:773]: ditto.

File: bin/resty (hunk 1 above)
Comment (syzh) [reply bin/resty:771]: The lua_socket_log_errors/lua_regex_cache_max_entries directive is not allowed here.
see: 
https://github.com/openresty/lua-nginx-module#lua_socket_log_errors
https://github.com/openresty/lua-nginx-module#lua_regex_cache_max_entries

**Comment by doujiang24** [comment]
@syzh Thanks for your contribution :)

**Pull request merged**